### PR TITLE
add some SEO

### PR DESCRIPTION
**Pull request by Crownss**

### CODE REVIEW

**Review by Zxce3 — APPROVED** [review]
Looks Good

**Review by Zxce3 — CHANGES_REQUESTED** [review]
I believe we shouldn't remove the yarn.lock file from the repository because doing so may cause issues with GitHub Actions. Please refer to https://github.com/maling-it/tools/actions/runs/7712790144 for more details.

File: src/components/CoreSwitcher.svelte
Comment (Zxce3) [inline src/components/CoreSwitcher.svelte:1]: whats the point?

File: src/app.html
Comment (Zxce3) [inline src/app.html:1]: Looks good tho

File: src/components/CoreSwitcher.svelte
Comment (Crownss) [reply src/components/CoreSwitcher.svelte:1]: auto format perhaps

**Comment by Crownss** [comment]
> I believe we shouldn't remove the yarn.lock file from the repository because doing so may cause issues with GitHub Actions. Please refer to https://github.com/maling-it/tools/actions/runs/7712790144 for more details.

i see thank you for your advice

**Review by Crownss — COMMENTED** [review]
remove yarn.lock from .gitignore

**Pull request merged**